manage.py

```diff
@@ -7,7 +7,7 @@
 
 def main():
     """Run administrative tasks."""
-    dotenv.read_dotenv()
+    dotenv.load_dotenv()
```

Comment:
why did you use dotenv.load_dotenv() ?

Reply:
okay @DamaMichaelYohanna. did you try installing independently to your virtual environment? e.g pip install django-dotenv

Reply:
Not sure about this one. Though I tried other suggestions from other resources.
Let me go ahead and try this one "pip install django-dotenv"

Reply:
@Teams-Motion the installation just worked. Let me run it and see the out come

Reply:
@Teams-Motion fixed now

Reply:
@DamaMichaelYohanna  okay so make the necessary changes and exclude the .idea folder before pushing. Your migration file had some issues try deleting it and make a new migration. then push the code again.

Reply:
Alright am at work right now. When am done I will make the necessary changes